Header header
##geometry_msgs/Pose pose
sensor_msgs/Image img_l
uint64 kf_id
